### ISSUE TITLE
Nearby share with a bunch of devices on the network

### ISSUE THREAD

**Issue by BlueDude11** [nearby-sharing/android#179]
So this is a work network and there are a lot of devices so all I'm asking for is a better list UI with a search option so we can search for a specific computer

**Comment by liang136146129** [comment]
Yes, there are too many devices on the list.  One more suggestion, add recently used device. It will be easier to find my device in the crowd.
